index.html

```diff
@@ -3397,7 +3270,84 @@ <h3>Device Theft and Impersonation</h3>
         </ul>
 
     </section>
+      <section>
+        <h2>Accessibility Impact</h2>
+        <p>There are a number of accessibility considerations of which implementors should be aware when processing data described in this specification. As with any implementation of web standards or protocols, ignoring accessibility issues will make this information unusable to a large subset of the population. It is important to follow accessiblity guidelines and standards such as [WCAG21] to ensure that all people, regardless of ability can make use of this data. This is especially important when establishing cryptography and encryption which historically have created problems for assistive technologies.
```

Comment:
If this was intentionally moved, can you restore the formatting to what was removed above?

Reply:
Actually, this also appears to be a reversion of another commit that moved the accessibility section: https://github.com/w3c/vc-data-model/commit/99b0dc99ce99878e63de6a188ea11edb6e6cdad3

Reply:
I will try to fix my mistakes now. Sorry for the finger trouble

Reply:
Can we simply delete this PR and I will start all over again with a new master copy